src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -261,6 +261,13 @@ conversion, so type inference fails because the type is not unique. Please note
 that you must write the `(())` in one sequence without intermediate whitespace
 so that `rustdoc` understands you want an implicit `Result`-returning function.
 
+## Showing warnings in doctests
+
+You can show warnings in doctests by running `rustdoc --test --test-args=--show-output`
+(or, if you're using cargo, `cargo test --doc -- --show-output`).
+By default, this will still hide `unused` warnings, since so many examples use private functions;
+you can add `#![warn(unused)]` to the top of your example if you want to see unused variables or dead code warnings.
```

Comment:
Could also mention `#![doc(test(attr(warn(unused))))]` in `lib.rs` to enable it globally.

Reply:
Sure, done.